third_party/parser/cc/grammars/typedruby.ypp

```diff
@@ -2061,6 +2061,10 @@ opt_block_args_tail:
                     {
                       $$ = driver.build.index(self, $1, $2, $3, $4);
                     }
+                | primary_value call_op
```

Comment:
@jez there's no such rule in any version of MRI. For now I think it's ok but it can potentially conflict with 2.6/2.7 changes. Is there any other way to handle it?

Reply:
`third_party/parser/cc/grammars/typedruby.ypp: warning: 85 shift/reduce conflicts`

Reply:
I'm not sure about bison, but at least in racc there's a way to get a stack in the `on_error` method (that's `yyerror` in bison):
``` ruby
# when parsing "A.new."
value_stack
 => [s(:send,
  s(:const, nil, :A), :new), [:dot, #<Parser::Source::Range (fragment:0) 5...6>]]
```

Reply:
Yeah i rolled this back; i solved my problems in a different way.